src/react/reactquestion_element.tsx

```diff
@@ -33,6 +33,17 @@ export class SurveyElementBase<P, S> extends React.Component<P, S> {
   componentDidUpdate(prevProps: any, prevState: any) {
     this.makeBaseElementsReact();
   }
+  private _canComponentUpdate = true;
+  protected get canComponentUpdate(): boolean {
+    return this._canComponentUpdate;
+  }
+  protected set canComponentUpdate(newValue: boolean) {
+    this._canComponentUpdate = newValue;
+    this.forceUpdate();
+  }
+  shouldComponentUpdate(nextProps:any, nextState:any):boolean {
```

Comment:
We have shouldComponentUpdate in SurveyQuestionElementBase.
SurveyQuestionElementBase will fully overrides this function.
Let's talk about it on Monday by voice. It is better to implement in another way.

Reply:
ok we migth support this functionality for SurveyQuestionElementBase too. Ok lets discuss.